gateway-ha/src/test/java/io/trino/gateway/ha/security/TestLbFilter.java

```diff
@@ -126,38 +141,41 @@ public void testSuccessfulHeaderAuthentication() throws Exception {
     lbFilter.filter(requestContext);
 
     // SecurityContext must be set with the right authorizer at authentication
-    Mockito.verify(requestContext, Mockito.times(1)).setSecurityContext(secContextCaptor.capture());
+    Mockito.verify(requestContext, Mockito.atLeast(1))
+            .setSecurityContext(secContextCaptor.capture());
 
     // Checks authorization for authenticated principal
-    Assert.assertTrue(secContextCaptor.getValue().isUserInRole("USER"));
-    Assert.assertTrue(secContextCaptor.getValue().isUserInRole("ADMIN"));
+    assertTrue(secContextCaptor.getValue().isUserInRole("USER"));
+    assertTrue(secContextCaptor.getValue().isUserInRole("ADMIN"));
 
   }
 
-  @Test(expected = WebApplicationException.class)
+  @Test
```

Comment:
how does the Junit 5 handle the expected exception?

Reply:
In Junit 5 you wrap the method in an `assetThrows`, I used a lambda to do this here